src/utils/src/api-utils.ts

```diff
@@ -98,4 +100,21 @@ export const routeMatcher = (
     }
   }
   return buildNotFoundResponse("route", event.path);
-  };
\ No newline at end of file
+};
+
+/**
+* Verifys a token and returns the payload/user if successful, otherwise returns the error
+* @param token The jwt token
+*/
+export const verifyToken = (
```

Comment:
How does the consumer of this function determine whether the output is a success vs an error? I think it might be better to ensure the consumer of the function either doesn't need to figure this out, or it is really easy for them to know

Reply:
I will change the function so it only returns the decoded payload if successful, and nothing if not. Consumers can then test if the return value is truthy it know there was not an error

Reply:
that should work